eiffel-vocabulary/EiffelServiceDiscontinuedEvent.md

```diff
@@ -19,7 +19,31 @@
 The EiffelServiceDiscontinuedEvent is to declare that a request to tear down a Service Instance has completed. The tear down of a service instance may be exectuted by a Cloud Orchestractor/scheduler.
```

Comment:
Is the request a "request" or "is allowed"?

Reply:
Apologies... I see the confusion.

How about replace the word to "a request" to "an activity" ?